### PR TITLE
added test cases for task Book

### PR DESCRIPTION
**Pull request by yuifuku1118**
Added some test cases that covers following classes
- JsonAdapterTask
- JsonSerializableTaskBook
- JsonTaskBookStorage

### CODE REVIEW

File: src/test/java/seedu/address/storage/JsonSerializableTaskBookTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import seedu.address.commons.util.JsonUtil;
+import seedu.address.model.TaskBook;
+import seedu.address.testutil.TypicalTasks;
+
+public class JsonSerializableTaskBookTest {
+
+    private static final Path TEST_DATA_FOLDER = Paths.get("src", "test", "data", "JsonSerializableTaskBookTest");
+    private static final Path TYPICAL_TASKS_FILE = TEST_DATA_FOLDER.resolve("typicalTasksTaskBook.json");
+    private static final Path INVALID_TASK_FILE = TEST_DATA_FOLDER.resolve("invalidTasksTaskBook.json");
+
```
Comment (ngchisern) [inline src/test/java/seedu/address/storage/JsonSerializableTaskBookTest.java:21]: Is it better to test the case where there are duplicate tasks too?
e.g. `DUPLICATE_TASK_FILE`

**Review by ngchisern — CHANGES_REQUESTED** [review]
Looks good. Added some minor suggestions below.

File: src/test/java/seedu/address/storage/JsonSerializableTaskBookTest.java (hunk 1 above)
Comment (yuifuku1118) [reply src/test/java/seedu/address/storage/JsonSerializableTaskBookTest.java:22]: Thanks for the suggestion!! Added that test case!

**Review by ngchisern — APPROVED** [review]
LGTM

**Pull request merged**